test/Confluent.Kafka.IntegrationTests/Tests/Tests.cs

```diff
@@ -14,21 +14,75 @@
 //
 // Refer to LICENSE for more information.
 
+using Confluent.Kafka;
+using Confluent.Kafka.Admin;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.IO;
 using System.Reflection;
 using Newtonsoft.Json.Linq;
-
+using Xunit;
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    public static partial class Tests
+    public class GlobalFixture : IDisposable
+    {
+        private string bootstrapServers;
+
+        public GlobalFixture()
```

Comment:
this `GlobalFixture` nonsense is required to implement lifecycle methods before and after a test set in Xunit (as opposed to before and after every test, which is easy to do).

Reply:
Suggest you add that as a comment, because there are no clues in the code.